usecase/parser/msg.go

```diff
@@ -143,17 +143,17 @@ func ParseBlockResultsTxsMsgToCommands(
 			case "/ibc.core.channel.v1.MsgTimeoutOnClose":
 				msgCommands = ibcmsg.ParseMsgTimeoutOnClose(msgCommonParams, txsResult, msgIndex, msg)
 			case "/cosmos.authz.v1beta1.MsgGrant":
-				// TODO: implement MsgGrant parser
+				msgCommands = ibcmsg.ParseMsgGrant(msgCommonParams, msg)
 			case "/cosmos.authz.v1beta1.MsgRevoke":
-				// TODO: implement MsgRevoke parser
+				msgCommands = ibcmsg.ParseMsgRevoke(msgCommonParams, msg)
 			case "/cosmos.authz.v1beta1.MsgExec":
-				// TODO: implement MsgExec parser
+				msgCommands = ibcmsg.ParseMsgExec(msgCommonParams, msg)
 			case "/cosmos.feegrant.v1beta1.MsgGrantAllowance":
-				// TODO: implement MsgGrantAllowance parser
+				msgCommands = ibcmsg.ParseMsgGrantAllowance(msgCommonParams, msg)
 			case "/cosmos.feegrant.v1beta1.MsgRevokeAllowance":
-				// TODO: implement MsgRevokeAllowance parser
+				msgCommands = ibcmsg.ParseMsgRevokeAllowance(msgCommonParams, msg)
 			case "/cosmos.vesting.v1beta1.MsgCreateVestingAccount":
-				// TODO: implement MsgCreateVestingAccount parser
+				msgCommands = ibcmsg.ParseMsgCreateVestingAccount(msgCommonParams, msg)
```

Comment:
Are these message really related to IBC? I think they are from different modules based on the type string.

Reply:
Right, will move them out of IBC namespace.